src/components/brightnessByColor.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Calculate brightness value by RGB or HEX color.
+ * @param color (String) The color value in RGB or HEX (for example: #000000 || #000 || rgb(0,0,0) || rgba(0,0,0,0))
+ * @returns (Number) The brightness value (dark) 0 ... 255 (light)
+ */
+const brightnessByColor = (color) => {
+  var color = '' + color, isHEX = color.indexOf('#') === 0, isRGB = color.indexOf('rgb') === 0;
+
+  if (isHEX) {
+  const hasFullSpec = color.length === 7;
+  var m = color.substr(1).match(hasFullSpec ? /(\S{2})/g : /(\S{1})/g);
```

Comment:
I put comments on these. var, m, r 
NEVER EVER use them. 

A professional software engineer does not declare a variable called m

Reply:
r, g, b here stand for RGB. changed the "m". Modified the function so that it doesn't need "var" anymore